.github/workflows/build_test.yml

```diff
@@ -17,40 +17,56 @@ on:
 jobs:
   ubuntu_build:
     name: Ubuntu Build ${{ matrix.name }}
-    runs-on: [ubuntu-latest]
+    runs-on: ${{ matrix.os }}
     strategy:
       matrix:
         # We have one job per "name" in the matrix. Attributes are set on the
         # specific job names.
-        name: [release, debug, asan, msan, scalar]
+        name: [release, debug, scalar, asan, msan, release-nojpeg, release-gcc8, release-clang-5]
         include:
           - name: release
+            os: ubuntu-latest
             test_in_pr: true
           - name: debug
+            os: ubuntu-latest
           # Build scalar-only hwy instructions.
           - name: scalar
+            os: ubuntu-latest
             mode: release
             cxxflags: -DHWY_DISABLED_TARGETS=~HWY_SCALAR
+          # TODO(eustas): why ASAN has no params?
```

Comment:
`mode` is the parameter we pass to ./ci.sh, and if mode is empty we use `name`.

asan, msan and debug don't need any parameters and that's why they are listed above. The rest don't need to be listed afaik.

Reply:
Got it. Should we remove then `release, msan, scalar` from list and empty `debug` expando?